src/test/java/ua/softserveinc/tc/validator/EmailValidatorTest.java

```diff
@@ -0,0 +1,119 @@
+package ua.softserveinc.tc.validator;
+
+/**
+ * Created by Valery on 16.11.2016.
+ */
+
+import org.junit.Assert;
+import org.junit.Test;
+import ua.softserveinc.tc.constants.ValidationConstants;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class EmailValidatorTest {
+
+
+    @Test
+    public void testForContainingADogSymbol() {
+        String incorrectEmailAddress = "mkyong";
+        Assert.assertEquals("Email address should contain a '@' symbol.",
+                false,
+                incorrectEmailAddress.matches(ValidationConstants.SIMPLE_EMAIL_REGEX));
```

Comment:
if matches returns booleas, you should ne fine with just assertTrue or assertFalse - this will be goon enought